pkgs/http/lib/src/browser_client.dart

```diff
@@ -53,6 +64,11 @@ class BrowserClient extends BaseClient {
 
   bool _isClosed = false;
 
+  /// Use different caching mode for a HTTP request.
+  /// Defaults to `default`.
+  // https://developer.mozilla.org/en-US/docs/Web/API/Request/cache
+  String cacheMode;
```

Comment:
This can be private and you can move the documentation to the constructor documentation.

Reply:
...and `final`!